unified-runtime/source/common/ur_pool_manager.hpp

```diff
@@ -248,6 +252,45 @@ pool_descriptor::create(ur_usm_pool_handle_t poolHandle,
   return {ret, descriptors};
 }
 
+inline std::pair<ur_result_t, std::vector<pool_descriptor>>
+pool_descriptor::createFromDevices(
```

Comment:
Can we use the same function for v2 adapter and remove `pool_descriptor::create` and `urGetAllDevicesAndSubDevices` functions?

Reply:
Done.